index.js

```diff
@@ -31,6 +31,10 @@ if (process.env.hasOwnProperty('GCLOUD_DEBUG_LOGLEVEL')) {
 if (process.env.hasOwnProperty('GCLOUD_DEBUG_DISABLE')) {
   config.enabled = false;
 }
+if (process.env.hasOwnProperty('GCLOUD_DEBUG_RELATIVE_REPOSITORY_PATH')) {
```

Comment:
I don't like env. var name too much, but I don't have obviously better suggestions at this point.